pkg/policy/apply.go

```diff
@@ -61,14 +61,12 @@ func applyPolicy(policy kyvernov1.PolicyInterface, resource unstructured.Unstruc
 		logger.Error(err, "failed to process mutation rule")
 	}
 
-	policyCtx := &engine.PolicyContext{
-		Policy:           policy,
-		NewResource:      resource,
-		ExcludeGroupRole: excludeGroupRole,
-		JSONContext:      ctx,
-		Client:           client,
-		NamespaceLabels:  namespaceLabels,
-	}
+	policyCtx := engine.NewPolicyContextWithJsonContext(ctx).
+		WithPolicy(policy).
+		WithNewResource(resource).
+		WithNamespaceLabels(namespaceLabels).
+		WithClient(client).
+		WithExcludeGroupRole(excludeGroupRole...)
```

Comment:
Missing `NamespaceLabels`.

Reply:
`WithNamespaceLabels(namespaceLabels).` is here, what is missing ?

Reply:
My bad🙈